src/genie/libs/parser/iosxe/show_stackwise_virtual.py

```diff
@@ -0,0 +1,110 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Optional
+
+
+# ===========================
+# Schema for:
+#  * 'show stackwise-virtual'
+# ===========================
+class ShowStackwiseVirtualSchema(MetaParser):
+    """Schema for show stackwise-virtual."""
+
+    schema = {
+        "domain": int,
+        "enabled": bool,
+        Optional("switches"): {
+            int: {
+                  "id": int,
+                  "ports": {
+                     int: str,
+                },
+            }
+        },
+    }
+
+
+# ===========================
+# Parser for:
+#  * 'show stackwise-virtual'
+# ===========================
+class ShowStackwiseVirtual(ShowStackwiseVirtualSchema):
+    """Parser for show stackwise-virtual"""
+
+    cli_command = ["show stackwise-virtual"]
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command[0])
+        else:
+            output = output
+
+        # Stackwise Virtual Configuration:
+        # --------------------------------
+        # Stackwise Virtual : Enabled
+        # Domain Number : 1  
+        # 
+        # Switch  Stackwise Virtual Link  Ports
+        # ------  ----------------------  ------
+        # 1       1                       TenGigabitEthernet1/0/47  
+        #                                 TenGigabitEthernet1/0/48  
+        # 2       1                       TenGigabitEthernet2/0/47  
+        #                                 TenGigabitEthernet2/0/48  
+
+        # Stackwise Virtual : Enabled
+        enabled_capture = "(?P<enabled>Enabled|Disabled)"
+        p_enabled = re.compile(f"Stackwise\s+Virtual\s+:\s+{enabled_capture}")
```

Comment:
We're afraid, but please change to `format()` instead of f-strings. pyATS still support python3.5 which doesn't support f-strings.